### PR TITLE
reformat all code using isort and flake8-quotes

### PR DESCRIPTION
**Pull request by Bogdanp**

### CODE REVIEW

File: dramatiq/__init__.py [hunk 1]
```diff
@@ -33,33 +38,40 @@
 
 __all__ = [
     # Actors
-    "Actor", "GenericActor", "actor",
-
+    "Actor",
```
Comment (rpkilby) [inline dramatiq/__init__.py]: You may want to consider one of the [multi line output modes](https://github.com/timothycrosley/isort#multi-line-output-modes), such as `multi_line_outuput=5`

Comment (Bogdanp) [reply dramatiq/__init__.py]: I was just looking at that but I don't think it applies to `__all__`. Also, I think the only mode that's compatible between black and isort is `3`.

Comment (rpkilby) [reply dramatiq/__init__.py]: Just to further throw in my two cents - I have mixed feelings about black's output. There are several style changes that seem like an improvement, but more that seem like they hurt readability. eg, changes to the argument parser, several method and test signatures, etc... idk if this is a result of black trying to format on a shorter line length.

¯\\\_(ツ)_/¯

Comment (Bogdanp) [reply dramatiq/__init__.py]: Yeah, I think I might drop black for now since it does hurt readability and it doesn't support Python 3.5.

Comment (rpkilby) [reply dramatiq/__init__.py]: In the mean time, flake8 has a number of extensions. eg, double quote enforcement could be handled with https://github.com/zheller/flake8-quotes

Comment (Bogdanp) [reply dramatiq/__init__.py]: Nice! I wasn't aware of that.

**Pull request merged**